certvalid/crlvalidator.go

```diff
@@ -18,19 +18,19 @@ type X509CRLValidator struct {
 	updateLock sync.RWMutex
 }
 
-/// Create a new validator using the specified CA and the CRL url specified
+// NewX509CRLValidator Create a new validator using the specified CA and the CRL url specified
 func NewX509CRLValidator(ca *x509.Certificate, crlURL string) *X509CRLValidator {
 	return &X509CRLValidator{
 		ca, nil, crlURL, time.Now().AddDate(0, 0, -1), sync.RWMutex{},
 	}
 }
 
-/// Get when the next validation call will fail because needs to call Update()
+// NextUpdateLimit Get when the next validation call will fail because needs to call Update()
 func (x *X509CRLValidator) NextUpdateLimit() time.Time {
 	return x.NextUpdate
 }
 
-/// Update the CRL, is sync safe, so it can be invocated within a goroutine
+// Update the CRL, is sync safe, so it can be invocated within a goroutine
 func (x *X509CRLValidator) Update() error {
```

Comment:
I'm not sure about these changes, however we would need to make some tests again with X509 certificates. 

Could you please add a TODO: x509 validation should be re-checked since an update on the API might have broken it

Reply:
Do you have a specific place to add this TODO? Or should I just add it in the beginning of the file?

Reply:
You can add it here on the Function comment.